RecoLocalMuon/RPCRecHit/interface/CSCSegtoRPC.h

```diff
@@ -6,10 +6,53 @@
 #include "DataFormats/RPCRecHit/interface/RPCRecHit.h"
 #include "DataFormats/RPCRecHit/interface/RPCRecHitCollection.h"
 
+class CSCStationIndex{
+public:
+    CSCStationIndex():_region(0),_station(0),_ring(0),_chamber(0){}
+    CSCStationIndex(int region, int station, int ring, int chamber):
+    _region(region),
+    _station(station),
+    _ring(ring),
+    _chamber(chamber){}
+    ~CSCStationIndex(){}
+    int region() const {return _region;}
+    int station() const {return _station;}
+    int ring() const {return _ring;}
+    int chamber() const {return _chamber;}
+    bool operator<(const CSCStationIndex& cscind) const{
+        if(cscind.region()!=this->region())
+            return cscind.region()<this->region();
+        else if(cscind.station()!=this->station())
+            return cscind.station()<this->station();
+        else if(cscind.ring()!=this->ring())
+            return cscind.ring()<this->ring();
+        else if(cscind.chamber()!=this->chamber())
+            return cscind.chamber()<this->chamber();
+        return false;
+    }
+    
+private:
+    int _region;
+    int _station;
+    int _ring;  
+    int _chamber;
+};
+
+
+class ObjectMapCSC{
+public:
+    ObjectMapCSC* GetInstance(const edm::EventSetup& iSetup);
+    std::set<RPCDetId> GetRolls(CSCStationIndex cscstationindex){return rollstoreCSC[cscstationindex];}
```

Comment:
Shouldn't GetRolls return a const reference? It would be faster. Every call to GetRolls I can find is in an assignment statement that copies the value into a new variable, so a const reference should work fine.